ReactAndroid/src/main/java/com/facebook/react/animated/NativeAnimatedNodesManager.java

```diff
@@ -353,9 +355,47 @@ public void onEventDispatch(Event event) {
    */
   public void runUpdates(long frameTimeNanos) {
     UiThreadUtil.assertOnUiThread();
+    boolean hasFinishedAnimations = false;
+
+    List<AnimatedNode> nodes = new ArrayList<AnimatedNode>(mUpdatedNodes.size() + mActiveAnimations.size());
+    for (int i = 0; i < mUpdatedNodes.size(); i++) {
+      AnimatedNode node = mUpdatedNodes.valueAt(i);
+      nodes.add(node);
+    }
+
+    for (int i = 0; i < mActiveAnimations.size(); i++) {
+      AnimationDriver animation = mActiveAnimations.valueAt(i);
+      animation.runAnimationStep(frameTimeNanos);
```

Comment:
This code didn't change, I just moved it in it's own method. How it works is we run the animation drivers first, which update the value nodes then update all the other nodes that are connected to these updated value nodes, which will cause props nodes to update their connected view.